standards/SPIFFE.md

```diff
@@ -34,7 +34,7 @@ An SVID itself is not a document type. Instead, we define 1) the properties requ
 The SPIFFE SVID is defined in the [SPIFFE Identity and Verifiable Identity Document](SPIFFE-ID.md) specification. The X.509 SVID specification is defined in [The X.509 SPIFFE Verifiable Identity Document](X509-SVID.md)
 
 ## 4. The Workload API
-The SPIFFE Workload API is the method through which workloads, or compute processes, obtain their SVID(s). It is typically exposed locally, and is unauthenticated. It is up to the implementor of the Workload API to authenticate the caller via an out-of-band method.
+The SPIFFE Workload API is the method through which workloads, or compute processes, obtain their SVID(s). It is typically exposed locally, and it is up to the implementor of the Workload API to authenticate the caller via an out-of-band method.
```

Comment:
The point we're trying to highlight here is that the workload need not do anything special, beyond hit this API, in order to get its SVID. At the same time, we wish to convey that we do expect the implementor of the Workload API to perform some sort of check in order to validate that the call is an authorized one. 

I think one problem with the text is that, `an out-of-band method` _could_ be construed as, the implementor of the workload API makes another call back into the original caller, requesting some sort of information. This is something we want to avoid, IMO.

I am struggling to come up with phrasing which gets this across... perhaps `s/out-of-band/indirect`? Or `an indirect, out-of-band method`?

Reply:
Maybe giving an explicit example would help, eg adding "One example would be to authenticate the caller via Unix credentials that are passed via the transport socket"? That was what made it clearer to me elsewhere

Although actually the traditional unix creds APIs require explicit creds passing, Linux is kind of unusual in that you can get implicit credentials from a domain socket, and I can imagine other situations where something has to be attached, even if callbacks are avoided. Process identity has not generally been a well defined concept in OSs.

Reply:
How about implementor can/must verify the authenticity of the caller.

Reply:
How about this? 

`It is typically exposed locally (eg. via a Unix Domain Socket), and does not explicitly support an authentication handshake or authenticating token from the workload. It is thus the responsibility of the implementor of the Workload API to verify the authenticity the caller via an out-of-band method, such as inspecting the properties of the process calling the Unix domain socket.`

Reply:
I think that sounds pretty good... perhaps `s/does not explicitly support/explicitly does not include/`?

Reply:
@evan2645 SGTM

@justincormack if this looks good to you, mind updating the PR and we'll get this approved. Appreciate your patience here (and contribution!)

Reply:
just my $0.02 
Instead of
>It is thus the responsibility of the implementor of the Workload API to verify the authenticity the caller via an out-of-band method

Replacing with 'can' or 'must' will make it look more specy
>Implementors can verify the authenticity of the caller to the Workload API via an out-of-band method

Reply:
heh, agreed with @pragashj... though MAY or SHOULD might be more appropriate.

Can suffices though, and I'm hesitant to introduce more back-and-forth on this semi-trivial matter. The forthcoming Workload API spec will further clarify, at which time we can rectify this doc if necessary